src/navigate-pop/tiling-stacking-windows.md

```diff
@@ -0,0 +1,86 @@
+# Manipulating Windows
+
+This section describes the Auto Tiling, Stacking, and window adjustment features in Pop!\_OS.
+
+---
+## Window Adjustment Mode
+
+Adjustment mode allows users to easily manipulate the placement of windows using keyboard shortcuts.
+
+Place a window into Management mode by positioning your cursor over the window, and then pressing `SUPER` + `Enter`. The window will be highlighted in yellow. You can now manipulate the window with `🠐` `🠑` `🠒` `🠓` keys in combination with other keys.
+
+![Adjustment Mode Manipulation](/images/tiling-stacking-windows/adjustment-mode-manipulation.gif)
+
+All keyboard shortcuts available in adjustment mode can be viewed and modified in Settings ➞ Keyboard ➞ View and Customize Shortcuts ➞ Move, resize, and swap windows.
+
+## Auto Tiling Windows
+
+Pop!\_OS optimizes workflows using a smart window management system called Auto Tiling. Auto Tiling automatically positions and sizes windows.
```

Comment:
I would expand the second sentence to explain _why,_ e.g. `Auto Tiling automatically positions and sizes windows to minimize wasted screen space.` or `Auto Tiling automatically positions and sizes windows to maximize screen utilization.`